docs/reference/docs/bulk.asciidoc

```diff
@@ -219,7 +219,10 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=source_excludes]
 
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=source_includes]
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=timeout]
+`timeout`::
+(Optional, <<time-units, time units>>) Time period to wait for
```

Comment:
The timeout is only used for waiting for active shards, auto-index creation (not relevant for delete though, that does not auto-create) or waiting for dynamic mapping updates. There is (currently) no timeout for regular indexing.